django/bossspatialdb/test/cutout_view_cloudvolume.py

```diff
@@ -0,0 +1,335 @@
+# Copyright 2021 The Johns Hopkins University Applied Physics Laboratory
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from django.conf import settings
+import blosc
+
+from rest_framework.test import APITestCase, APIRequestFactory
+from rest_framework.test import force_authenticate
+from rest_framework import status
+
+from bossspatialdb.views import Cutout
+
+from bosscore.test.setup_db import SetupTestDB
+from bosscore.error import BossError
+
+import numpy as np
+import zlib
+import io
+import time
+from PIL import Image
+from cloudvolume import CloudVolume
+
+
+# S3 Bucket for CloudVolume Testing.
+TEST_BUCKET = "bossdb-test-data"
+
+# bossDB API Version
+version = settings.BOSS_VERSION
+
+class CutoutInterfaceViewCloudVolumeMixin(object):
+    def test_channel_uint8_cuboid_aligned(self):
+        "uint8 - cuboid aligned - no offset"
+        # Generate random data
+        test_mat = np.random.randint(1, 254, (16, 128, 128))
+        test_mat = test_mat.astype(np.uint8)
+
+        # Upload Data to Cloudvolume
+        self.vol_uint8[0:128, 0:128, 0:16] = test_mat.T
+
+        # Create Request to get data you posted
+        factory = APIRequestFactory()
+        request = factory.get('/' + version + '/cutout/col1/exp1/chan1/0/0:128/0:128/0:16/',
+                              accepts='application/blosc')
+
+        # log in user
+        force_authenticate(request, user=self.user)
+
+        # Make request
+        response = Cutout.as_view()(request, collection='col1', experiment='exp1', channel='chan1',
+                                    resolution='0', x_range='0:128', y_range='0:128', z_range='0:16', t_range=None).render()
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        # Decompress
+        raw_data = blosc.decompress(response.content)
```

Comment:
cvdb handles the `brotli` stuff since it downloads using cloudvolume and cloudvolume returns raw numpy array. I then compress it using blosc to maintain compatibility with existing endpoints and renderers